docs/modules/optimisation.md

```
@@ -154,4 +154,8 @@ The `run_policy_optimisation` function returns a completed Optuna `study` object
 
 You can also access the `user_attrs` of the best trial to see the values of all other metrics for that trial, which is useful for understanding trade-offs.
 
-For more advanced analysis, Optuna provides powerful visualization tools (e.g., `optuna.visualization.plot_optimization_history`) that can be used with the returned `study` object.
\ No newline at end of file
+<<<<<<< HEAD
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)